src/MarginTrading.AssetService.Contracts/Products/DiscontinuedProduct.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace MarginTrading.AssetService.Contracts.Products
+{
+    public class DiscontinuedProduct
```

Comment:
Shouldn't it be 
```suggestion
    public class DiscontinuedProductContract
```
or even 
```suggestion
    public class DiscontinuedProductIdContract
```
since you are not exposing the whole product contract?